src/utils/index.ts

```diff
@@ -143,3 +143,7 @@ export const checkAndMergeConnections = (
   }
   return connectionsToMerge
 }
+
+export const caseInsensitiveIncludes = (arr: string[], s1: string): boolean =>
```

Comment:
Do we want to use the built in `includes` function here? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes